utilities/list-helpers.metta

```diff
@@ -74,6 +74,18 @@
 (= (extend Nil $ys) $ys)
 (= (extend (Cons $x $xs) $ys) (Cons $x (extend $xs $ys)))
 
+; a function to merge two tuples (A B C) (D E F) ==> (A B C D E F)
```

Comment:
Better if you move this function below the  `++` function. The `++` function is similar to `extend` so it's better if they are together :)

Reply:
Can you put a type definition for the `concatTuple` function too?